scripts/decode_ui.gd

```diff
@@ -16,11 +16,6 @@ const COSTS = {
 
 func _ready() -> void:
 	hide()
-	update_points()
-
-func update_points() -> void:
```

Comment:
Hebben we hier een nieuwe rendering voor of gaat dat toegevoegd moeten worden (ik gebruik deze momenteel in de deciphershop)